pytorch_lightning/core/lightning.py

```diff
@@ -519,8 +534,10 @@ def training_step(self, *args, **kwargs) -> STEP_OUTPUT:
                 The output of your :class:`~torch.utils.data.DataLoader`. A tensor, tuple or list.
             batch_idx (int): Integer displaying index of this batch
             optimizer_idx (int): When using multiple optimizers, this argument will also be present.
-            hiddens(:class:`~torch.Tensor`): Passed in if
-                :paramref:`~pytorch_lightning.trainer.trainer.Trainer.truncated_bptt_steps` > 0.
+            hiddens(:class:`~torch.Tensor`): Passed in if either
+                :paramref:`~pytorch_lightning.core.lightning.LightningModule.truncated_bptt_steps` > 0
+                :paramref:`~pytorch_lightning.trainer.trainer.Trainer.truncated_bptt_steps` > 0
```

Comment:
Let's not mention the deprecated option?

```suggestion
            hiddens(:class:`~torch.Tensor`): Passed in if
                :paramref:`~pytorch_lightning.core.lightning.LightningModule.truncated_bptt_steps` > 0.
```